### PR TITLE
truck list created by fetching data from api

### PR DESCRIPTION
**Pull request by Atharv-Joshi**

### CODE REVIEW

File: lib/widgets/myTrucksCard.dart [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+                    ),
+                  ),
+                  Text(
+                      'Available',
+                      style: TextStyle(
+                        fontSize: size_11),
+                  ),
+                ],
+              ),
+              Container(
```
Comment (liveasy97) [inline lib/widgets/myTrucksCard.dart:65]: remove this container

File: lib/screens/myTrucks.dart [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+        padding: EdgeInsets.fromLTRB(space_4, space_8, space_4, space_4),
+        child: Column(
+          children: [
+            Row(
+               mainAxisAlignment: MainAxisAlignment.spaceBetween,
+              children: [
+                Row(
+                  mainAxisAlignment: MainAxisAlignment.start,
+                  children: [
+                    BackButtonWidget(),
```
Comment (liveasy97) [inline lib/screens/myTrucks.dart:42]: you can remove this back button from here, its non fuctional here

File: lib/widgets/myTrucksCard.dart [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+                                fontSize: size_9),
+                          ),
+                          Container(
+                            margin: EdgeInsets.only(top: space_3),
+                            child: Text(
+                              'Driver',
+                              style: TextStyle(
+                                  fontSize: size_8),),
+                          ),
+                          Text(
```
Comment (liveasy97) [inline lib/widgets/myTrucksCard.dart:132]: try getting this name from driverId

File: lib/screens/myTrucks.dart (hunk 2 above)
Comment (Atharv-Joshi) [reply lib/screens/myTrucks.dart:42]: ok

File: lib/widgets/myTrucksCard.dart (hunk 1 above)
Comment (Atharv-Joshi) [reply lib/widgets/myTrucksCard.dart:65]: but it is used to give margin
 to text widget?

File: lib/widgets/myTrucksCard.dart (hunk 3 above)
Comment (Atharv-Joshi) [reply lib/widgets/myTrucksCard.dart:132]: will work on this

File: lib/widgets/myTrucksCard.dart (hunk 1 above)
Comment (liveasy97) [reply lib/widgets/myTrucksCard.dart:65]: there is no need to show varanasi here thats why I asked to remove the container.

Comment (Atharv-Joshi) [reply lib/widgets/myTrucksCard.dart:65]: ok

**Review by liveasy97 — APPROVED** [review]

**Pull request merged**